Periodic_3_triangulation_3/demo/Periodic_3_triangulation_3/Scene.cpp

```diff
@@ -1463,7 +1464,15 @@ void Scene::gl_draw_conflict() {
     std::vector<Facet> boundary_facets;
     // Find the conflict region
     Cell_handle c = p3dt.locate(moving_point);
-    p3dt.find_conflicts(moving_point,c,std::back_inserter(boundary_facets),std::back_inserter(cic),CGAL::Emptyset_iterator());
+    Point _a(c->vertex(0)->point()),
+          _b(c->vertex(1)->point()),
+          _c(c->vertex(2)->point()),
+          _d(c->vertex(3)->point());
+    if(!( moving_point == _a
+       || moving_point == _b
+       || moving_point == _c
+       || moving_point == _d))
+      p3dt.find_conflicts(moving_point,c,std::back_inserter(boundary_facets),std::back_inserter(cic),CGAL::Emptyset_iterator());
```

Comment:
You are redoing part of the job of `locate()`, here.

Instead, use the [full locate with the `Locate_type`][1], and verify that the locate type is not `VERTEX`.

[1]: https://doc.cgal.org/4.13/Periodic_3_triangulation_3/classCGAL_1_1Periodic__3__triangulation__3.html#ae0e1ff13ddae388a580212f2645886ac